infrastructure-scripts/claude_mcp.sh

```
@@ -0,0 +1,700 @@
+cleanup_failed_server() {
+    local name="$1"
+    echo -e "${YELLOW}  🧹 Cleaning up failed installation of $name...${NC}"
+    claude mcp remove "$name" >/dev/null 2>&1 || true
+}
+
+# Enhanced function to add MCP server with full error checking
+add_mcp_server() {
+    local name="$1"
+    local package="$2"
+    shift 2
+    local args="$@"
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)